cfme/tests/cloud_infra_common/test_genealogy.py

```diff
@@ -87,6 +88,10 @@ def _():
             parent = pytest.sel.text(opt).strip()
         assert parent.startswith(small_template), "The parent template not detected!"
     else:
-        vm_crud_ancestors = vm_crud.genealogy.ancestors
+        try:
+            vm_crud_ancestors = vm_crud.genealogy.ancestors
+        except NameError:
+            logger.exception("The parent template not detected!")
+            raise pytest.fail("The parent template not detected!")
```

Comment:
OPTIONAL: Would it make sense to log the name of the small template here as well? I'm not 100% sure about the context here - would it make it easier to debug?